echo_api.rb

```diff
@@ -41,6 +45,21 @@ def echo_response
     body = Base64.encode64(body)
   end
 
+  # Return all request headers like
+  #   ECHO.*: <foo>: <bar> as a response header <foo>: <bar>
+  #   ECHO.*: <baz>        as a response header <baz>:
+  get_echoable_headers.each do |h|
```

Comment:
IMO it would be much clearer by expanding the values like:
```ruby
echoable_headers.each do |(key, value)|
...
```

to get rid of the `h[1]` and give it better name